### ISSUE TITLE
Refactor operation emissions script to use dicts instead of lists

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#315]
This is sample code from the operation emissions script:

```
# for heating services
    heating_services = [[Qhs_flag, 'Qhsf_MWhyr', 'Qhsf', 'Af_m2']]
    for x in heating_services:
        fields_to_plot = ['Name', x[2] + '_pen_GJ', x[2] + '_ghg_ton', x[2] + '_pen_MJm2', x[2] + '_ghg_kgm2', 'GFA_m2']
        heating[fields_to_plot[1]] = heating[x[1]] * heating['PEN'] * 3.6
        heating[fields_to_plot[2]] = heating[x[1]] * heating['CO2'] * 3.6
        heating[fields_to_plot[3]] = heating[x[1]] * heating['PEN'] * 3600/heating['GFA_m2']
        heating[fields_to_plot[4]] =  heating[x[1]] * heating['CO2'] * 3600/heating['GFA_m2']
        if x[0]:
            heating[fields_to_plot].to_csv(os.path.join(result_folder, '%s_LCA_operation.csv' % x[2]), index=False,
                                                        float_format='%.2f')
```

This is hard to follow, since the `heating_services` list contains a list of "templates" that are then looped over. But the templates actually use positions to specify information... Instead, I propose to use a dictionary - just as quick and dirty, but easier to read:

```
heating_services = [{'write_to_file': Qhs_flag, 'source_field': 'Qhsf_MWhyr', 'prefix': 'Qhsf'}]
```

The `'Af_m2'` part is never used, so we can remove it.
This makes the code easier to read, because you have stuff like:

```
    if x['write_to_file']:
            heating[fields_to_plot].to_csv(os.path.join(result_folder, '%s_LCA_operation.csv' % x['prefix']), index=False,
                                                        float_format='%.2f')
```

or

```
fields_to_plot = ['Name', x['prefix'] + '_pen_GJ', x['prefix'] + '_ghg_ton', x['prefix'] + '_pen_MJm2', x['prefix'] + '_ghg_kgm2', 'GFA_m2']
```

or 

```
heating[fields_to_plot[1]] = heating[x['source_field']] * heating['PEN'] * 3.6
```

**Comment by JIMENOFONSECA** [comment]
we won't have this
